var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -171,6 +171,8 @@ class PyScipy(PythonPackage):
     conflicts("%gcc@:7", when="@1.10:", msg="SciPy 1.10-1.13 requires GCC >= 8.0")
     conflicts("%gcc@:9.0", when="@1.14:", msg="SciPy 1.14: requires GCC >= 9.1")
     conflicts("%gcc@:4.7", when="@:1.9", msg="SciPy requires GCC >= 4.8")
+    # https://github.com/spack/spack/issues/48243
+    conflicts("%intel", when="@1.14:", msg="SciPy 1.14: Use Intel LLVM instead of Intel Classic")
```

Comment:
Let's move this below gcc/clang/msvc, the above documentation link is for all 3 of those

Reply:
Sure, will do momentarily